models/timm/beit_base_patch16_384.py

```diff
@@ -0,0 +1,19 @@
+# labels: name::beit_base_patch16_384 author::timm
```

Comment:
Is the only difference between this model and `beit_base_patch16_224` that this one uses image size 384x384 instead of 224x224?

If so, is it possible to move the image size selection to a `mlagility.parse()` command since there's potentially only ~100 unique models in here not 800+? Or is the image size a property of the model itself?

Reply:
> Is the only difference between this model and `beit_base_patch16_224` that this one uses image size 384x384 instead of 224x224?

Apart from the input size, the weights of those models are also **should**  be different if we set `pretrained = True`. Therefore, merging models might become problematic if we ever want to use the models with their weights to check for accuracy.

We now have exactly one file for each model in `timm.list_models()`. Some of those are the same model with a different `default_cfg`. Merging models should be possible if we don't care about keeping that 1-1 ratio between `timm.list_models()` and our python files (and if we don't care about the weights).

Reply:
People keep asking about accuracy tests... so let's take the position that we do care about the weights. Thanks for the explanation, resolving!